Extension.php

```diff
@@ -15,6 +15,46 @@ public function initialize()
         $this->app->before(array($this, 'before'));
 
         /*
+         * Extension config file:
+         * You extension can have its own config file where your users can set some options
+         * through the Bolt backend or any text editor.
+         * Look at the file named config.yml.dist which is your config boilerplate.
+         * You can edit it like you want so it fits your needs.
+         * Bolt will copy it as {extension-name}.{vendor-name}.yml into /app/config/extensions when it doesn’t already exist there.
+         * See also the section in the docs: https://docs.bolt.cm/extensions/essentials#using-your-own-configyml-file
+         * */
+
+        // dump the whole config on top of each page for debugging purpose
+        //dump($this->config);
+
+        // it's just a normal array
+        $foo = $this->config['foo'];
+
+
+
+
+
```

Comment:
Better for the overview. Can short it to just 4 lines in the next PR when it is too much.